.github/workflows/smoke-codespace.yaml

```diff
@@ -19,6 +19,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
+    - name: Maximize build space
+      uses: easimon/maximize-build-space@master
+      with:
+        remove-dotnet: 'true'
+        remove-android: 'true'
+        remove-haskell: 'true'
+
```

Comment:
Should we keep this?

Reply:
If this isn't helping with the test failures, then it would be nice to remove.